timer/src/main/java/com/toggl/timer/log/ui/TimeEntriesLogFragment.kt

```diff
@@ -96,6 +97,7 @@ class TimeEntriesLogFragment : Fragment(R.layout.fragment_time_entries_log) {
                 .distinctUntilChanged()
                 .onEach { isEditViewExpanded ->
                     if (isEditViewExpanded) {
+                        requireContext().performClickHapticFeedback()
```

Comment:
I don't think we need to `requireContext` for an action that's so non-critical like a little vibration
```suggestion
                context?.performClickHapticFeedback()
```